WasteManagement/WasteContainer/doc/spec.md

```diff
@@ -33,6 +33,19 @@ to keep backwards compatibility with current FIWARE reference implementations.
     + Allowed values: Interval [0,1]. 
     + Optional
     
++ `weight` : Weight of the container load.
+    + Attribute type: [Number](http://schema.org/Number)
```

Comment:
are you sure we are going to be able to know the weight of the container load or just the load of all the set i.e. container + its load ?